pkg/dock/proto/dock.proto

```diff
@@ -58,6 +58,9 @@ service Dock {
     
     // Delete a volume attachment
     rpc DeleteAttachment (DeleteAttachmentOpts) returns (GenericResponse){}
+
+    // Create a replication
+    rpc CreateReplication(CreateReplicationOpts) returns (GenericResponse) {}
```

Comment:
You missed a blank letter here, is that the reason why the protobuf failed to generate the DockClient Interface?

Reply:
Added it back and tried again. Still not generating.

Reply:
What protobuf version are you using?

Reply:
I used to use v2.3 I guess, but I have upgraded it to 2.7

Reply:
I think 2.7 is too old.  I had 2.x before but it says it need 3.x.